apps/passport-client/components/shared/PCDCard.tsx

```diff
@@ -269,9 +265,7 @@ const TicketWrapper = forwardRef<
       ? `${window.location.origin}/#/checkin-by-id`
       : ticketCategory === TicketCategory.ZuConnect
       ? `${window.location.origin}/#/verify`
-      : ticketCategory === TicketCategory.Generic
-      ? `${window.location.origin}/#/generic-checkin`
-      : undefined;
+      : `${window.location.origin}/#/generic-checkin`;
```

Comment:
I think this link won't work for anything that isn't Devcon.

My understanding is that we want to show QR codes for Devcon tickets, but not for anything else. Non-Devcon tickets should show either:

- The `imageUrl` for the ticket, if it has one
- A placeholder image if the ticket does not have an `imageUrl`

Our original code for this is a bit messy, but I think we can simplify this down to something like `if (isDevconTicket) { showQrCode() } else { showImage(} }`.

Reply:
i reverted this change and did something similar to what you said, only if there is a `idBasedVerifyURL` show the qr.

Reply:
I think there should not be any QR codes at all, even for tickets with `idBasedVerifyURL`.

The QR codes point to URLs that will not exist in the new UI, so they should be removed.

Reply:
@robknight agreed -- I don't think it's blocking this PR (idBasedVerifyURL will just get redirected to zupass.org, essentially), but logged here

https://linear.app/0xparc-pcd/issue/0XP-1474/remove-idbasedverifyurl-and-replace-with-a-placeholder-image